install.sh

```diff
@@ -69,9 +69,15 @@ set -euo pipefail
   fi
   echo "bin_path=$bin_path"
 
+  if [[ -n "$version" ]]; then
+    release="tags/${version}"
+  else
+    release="latest"
+  fi
```

Comment:
```suggestion
  fi
  echo "release=$release"
```
might as well also print that back out for feedback